app/lib/features/chat/utils.dart

```diff
@@ -376,3 +377,17 @@ Future<void> saveDraft(
     }
   }
 }
+
+Future<void> onMessageLinkTap(Uri uri, BuildContext context) async {
+  final roomId = getRoomIdFromLink(uri);
+
+  ///If link is type of matrix room link
+  if (roomId != null) {
+  }
```

Comment:
I understand this is a copy-paste thing, but could we maybe fix this? 

1. not have the closing `}` of the `if` in a different line than the `else {` (and several lines in between)
2. have an empty `if` just for an else ... that should be if an if checking the opposite
3. do something in the other case. like we want to redirect to the room, don't we?

Reply:
You mean like this suggested code:
```
Future<void> onMessageLinkTap(Uri uri, BuildContext context) async {
  final roomId = getRoomIdFromLink(uri);

  ///If link is type of matrix room link
  if (roomId != null) {
    goToChat(context, roomId);
  }

  ///If link is other than matrix room link
  ///Then open it on browser
  if (roomId == null) {
    await openLink(uri.toString(), context);
  }
}
```

Reply:
else the second part should be `} else { ` but other than that, yeah!